client/profile-wizard/index.js

```diff
@@ -150,15 +150,18 @@ class ProfileWizard extends Component {
 		return currentStep;
 	}
 
-	async goToNextStep() {
+	/**
+	 * @param {string} trackStepKey optional track key for steps that have multiple tabs containing their own steps.
+	 */
+	async goToNextStep( trackStepKey = '' ) {
```

Comment:
I like this solution, but I think `trackStepKey` may blur the lines of the logic around what is considered a "step" or a component in a step.

What if we pass an option `tracksArgs = {}` here that keeps this solution open-ended for other data and keeps our step name consistent?

Reply:
Yeah that works as well, and I guess I can make sure that `tracksArgs` can overwrite the `step` attribute in this case.

Reply:
Updated here: https://github.com/woocommerce/woocommerce-admin/pull/8265/commits/e3bd8b9ba43aac57d127c6db64f6a5d327e3ff70